src/config/gremlins/gremlins.tsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 import type { GremlinList } from '../../state';
 
-export const gremlins: GremlinList = {
-  3: {
+export type GremlinId =
+  | 'GREMLIN_MANAGEMENT_YELLS'
+  | 'GREMLIN_EMERGENCY_ON_OTHER_TEAM'
+  | 'GREMLIN_NOT_PULLING_THEIR_WEIGHT'
+  | 'GREMLIN_NOT_AT_DAILY_SCRUM';
+
+export const gremlins: GremlinList<GremlinId> = {
+  GREMLIN_MANAGEMENT_YELLS: {
     name: 'Management yells at a team member in public',
+    probability: () => 10,
```

Comment:
i was not sure what is needed to calculate probability. Currently `GameState` is passed. Let me know if you need s.th. else.